kits/api-breaking-change-detector/samples/test_flow.py

```
@@ -0,0 +1,57 @@
+    
+    # 2. Add timeout parameter (connect timeout: 10s, read timeout: 60s)
+    response = requests.post(API_URL, json=payload, headers=headers, timeout=(10, 60))
+    
+    # 3. Throw HTTP error exception if status code is not 200 OK
+    response.raise_for_status()
+    
+    res_data = response.json()
+    
+    # Handle GraphQL / Lamatic workflow response errors cleanly
+    if "errors" in res_data:
+        raise RuntimeError(f"Workflow execution failed with errors: {res_data['errors']}")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)